src/main/resources/application.properties

```diff
@@ -53,38 +49,38 @@ spring.data.mongodb.database: eiffel_intelligence
 # spring.data.mongodb.username:
 # spring.data.mongodb.password:
 # Configure the session storage expiration time for logged in users of Eiffel Intelligence front-end
-server.session-timeout: 1200
+server.session.timeout: 1200
 # Configure the database collection name to store sessions
 sessions.collection.name: sessions
 
 # Name of the database collection where aggregated objects are stored
-aggregated.collection.name: aggregated_objects
+aggregations.collection.name: aggregations
 # Configure time-to-live value (seconds) for aggregated objects. This needs to be set if
 # using the example rule set 'AllEventsRules' to avoid the storage growing too large.
 # Recommended setting is 10 minutes.
-aggregated.collection.ttlValue:
+aggregations.collection.ttl:
 # The name of the database collection where to store the mapping between
 # aggregated objects and id of the Eiffel events that contributed to those
-event_object_map.collection.name: event_object_map
+event.object.map.collection.name: event_object_map
 # Name of the database collection where subscriptions are stored
-subscription.collection.name: subscription
+subscriptions.collection.name: subscriptions
 # Name of the database collection where we store pair of 'subscription:aggregated object'
 # where notification has been sent and should not be repeated
-subscription.collection.repeatFlagHandlerName: subscription_repeat_handler
+subscriptions.repeat.handler.collection.name: subscriptions_repeat_handler
 # The waitlist contains Eiffel events which are waiting to be aggregated
-# Configure the database collection name for the waitlist
+# Configure the database collection name for the wait list
```

Comment:
If we have waitlist put together in the properties shouldn't we be consistent and use it as one word everywhere?

Reply:
It's a bit "huller om buller" and I think I've seen written with a space in several places. I will try to find them and put them together.